src/templates/location.tsx

```diff
@@ -1,6 +1,7 @@
 
-import { graphql } from 'gatsby'
+import { graphql, Link } from 'gatsby'
 import { useContext, useEffect, useState } from 'react'
+import { Breadcrumb } from 'react-bootstrap'
```

Comment:
Is this used?

Reply:
Nope, probably from a typo when I was making my changes and I missed it when I did my compare. I just made a commit that removes it.